.github/workflows/cibuildwheel.yml

```diff
@@ -28,7 +40,7 @@ jobs:
 
       - name: Install cibuildwheel
         run: |
-          python -m pip install cibuildwheel==1.6.3
+          python -m pip install cibuildwheel
```

Comment:
It's a bit safer to keep this pinned, we try not to break things, but you don't want any nasty surprises right before a release! How about using the action, perhaps?